packages/core/src/components/breadcrumb/breadcrumb.scss

```diff
@@ -186,7 +186,7 @@
   @include breadcrumb('ghost');
 }
 
-.dropdown-overflow .dropdown-menu {
-  max-height: 25rem;
```

Comment:
The dropdown will still grow past the screen height if it contains too many items.

Reply:
max height should something like `70vh`